ci-scripts/e2e-cypress.sh

```diff
@@ -73,6 +70,6 @@ fi
 echo '-----'
 echo "Running Cypress end to end tests"
 
-yarn e2e:cy"${INTEGRATION}":start-run-ci"${CI_ENV}${SUITE}"
+yarn e2e${INTEGRATION}:start:run:ci"${SUITE}"
```

Comment:
Isn't suite and integration treated the same now? Only last part of the command?

Reply:
Yup. Integration removed.